kubeadm/ansible/group_vars/hosts.yml

```diff
@@ -0,0 +1,93 @@
+---
+all:
+  children:
+    # LOCAL
+    local:
+      hosts:
+        local0:
+          ansible_connection: local
+    # CLUSTER NODES
+    allnodes:
+      children:
+        # MASTER NODES
+        masters:
+          children:
+            # MASTER 1
+            # TODO: Generify to allow for X amount of masters
+            mainMaster:
+              hosts:
+                master0:
+                  ansible_host: 172.26.22.120
+                  name: 172.26.22.120
+                  initial_advertise_peer_url: https://172.26.22.120:2380
+                  listen_peer_url: https://172.26.22.120:2380
+                  listen_client_urls:
+                    - https://172.26.22.120:2379
+                    - http://127.0.0.1:2379
+                  advertise_client_url: https://172.26.22.120:2379
+            # SECONDARY MASTERS
+            secondaryMasters:
+              hosts:
+                master1:
+                  ansible_host: 172.26.22.121
+                  name: 172.26.22.121
+                  initial_advertise_peer_url: https://172.26.22.121:2380
+                  listen_peer_url: https://172.26.22.121:2380
+                  listen_client_urls:
+                    - https://172.26.22.121:2379
+                    - http://127.0.0.1:2379
+                  advertise_client_url: https://172.26.22.121:2379
+                master2:
+                  ansible_host: 172.26.22.122
+                  name: 172.26.22.122
+                  initial_advertise_peer_url: https://172.26.22.122:2380
+                  listen_peer_url: https://172.26.22.122:2380
+                  listen_client_urls:
+                    - https://172.26.22.122:2379
+                    - http://127.0.0.1:2379
+                  advertise_client_url: https://172.26.22.122:2379
+          # VARIABLES FOR MASTER NODES
+          vars:
+            pod_subnet: "100.96.0.0/11"
+            apiserver_count: "3"
+        # WORKER NODES
+        workers:
+          hosts:
+            worker0:
+              ansible_host: 172.26.22.100
+              name: vgtk8worker01
```

Comment:
Maybe we should change this to worker1 or something more generic?

Reply:
Good catch.